src/components/buttons/export-button.cjsx

```diff
@@ -0,0 +1,89 @@
+BS = require 'react-bootstrap'
+React = require 'react'
+BindStoreMixin = require '../bind-store-mixin'
+Time = require '../time'
+AsyncButton = require './async-button'
+
+{PerformanceExportStore, PerformanceExportActions} = require '../../flux/performance-export'
+
+module.exports = React.createClass
+  displayName: 'ExportPerformance'
+  propTypes:
+    courseId: React.PropTypes.string.isRequired
+    className: React.PropTypes.string
+
+  mixins: [BindStoreMixin]
+  bindStore: PerformanceExportStore
+
+  getInitialState: ->
+    downloadUrl: null
+    lastExported: null
+
+  isUpdateValid: (id) ->
+    {courseId} = @props
+    id is courseId
+
+  componentWillMount: ->
+    {courseId} = @props
+    PerformanceExportActions.load(courseId)
```

Comment:
Any reason for not using `<Loadable>` here instead of attaching the listeners manually?